types/gl-react-dom/index.d.ts

```diff
@@ -4,14 +4,13 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import * as React from 'react';
 import * as glReact from 'gl-react';
+import { GLViewDOM, GLViewDOMProps } from './GLViewDOM';
 
-export interface SurfaceProps extends glReact.SurfaceProps {
-  width: number;
-  height: number;
-}
+export interface SurfaceProps extends glReact.SurfaceProps, GLViewDOMProps {}
 
-export class Surface extends glReact.Surface {
+export class Surface extends glReact.Surface<GLViewDOM> {
   props: SurfaceProps;
 }
+
+export {};
```

Comment:
This line is unnecessary.
In order to have an ES6 style, callable import statement when consuming these type definitions, the consumer must enable `--esModuleInterop` (or more specifically, `--allowSyntheticDefaultImports`), thereby allowing them to use `import NetlifyIdentityWidget from "netlify-identity-widget";` in the same way that one would use `import NetlifyIdentityWidget = require("netlify-identity-widget");`

Reply:
The `export {};` was left unintentionally, since I decided to move internal types to a separate file (and to not to re-export them on `index.d.ts`). The `export {};` was just a try to get off from tslint warning that "everything is exported". So it's no removed.